src/components/ListComposable/components/ListProvider/useSelectedState.ts

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import type {ListItemId} from '../../types';
+
+export const useSelectedState = (_selected?: Record<ListItemId, boolean>) => {
```

Comment:
Could we use [this](https://github.com/gravity-ui/uikit/blob/b9bda434a8a3722c088758efbf990f63cd221e62/src/components/utils/useSelect/useSelect.ts) hook instead?